v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/templates/ProtegrityDataTokenization.java

```diff
@@ -164,5 +183,51 @@ private static WriteResult writeToBigQuery(PCollection<Row> input, String bigQue
         }
         return failsafeElement;
     }
+    private static PDone writeToBigTable(
+            PCollection<Row> input,
+            String bigTableProjectId,
+            String bigTableInstanceId,
+            String bigTableTableId,
+            Schema schema
+    ) {
+        return input.apply("convertToBigTableFormat", ParDo.of(new TransformToBigTableFormat(schema)))
+                .apply("writeToBigTable", BigtableIO.write()
+                        .withProjectId(bigTableProjectId)
+                        .withInstanceId(bigTableInstanceId)
+                        .withTableId(bigTableTableId));
+    }
+
+    static class TransformToBigTableFormat extends DoFn<Row, KV<ByteString, Iterable<Mutation>>> {
```

Comment:
Is there any way to make this transform and/or writing to BT failsafe?

Reply:
I think making this failsafe should be in a separate PR to reduce PR size and possible merge conflicts

Reply:
Sounds fair, please, take care of this after merging this PR.